tests/test_routes.py

```diff
@@ -99,6 +101,15 @@ def test_upload_xlsx_validation_errors(requests_mock, example_pre_ingest_data_fi
     assert "You are missing project locations. Please enter a project location." in str(page_html)
     assert "Start date in an incorrect format. Please enter a dates in the format 'Dec-22'" in str(page_html)
 
+    # assert errors display correctly
+    assert '<th class="govuk-table__header" scope="col">Description</th>' in str(page_html)
+    assert '<th class="govuk-table__header" scope="col">Cell</th>' in str(page_html)
+    assert (
+        '<td class="govuk-table__cell">You are missing project locations. Please enter a project location.</td>'
+        in str(page_html)
+    )
+    assert '<td class="govuk-table__cell">Section2</td>' in str(page_html)
+
```

Comment:
I don't think we need to make these extra assertions as they're just testing the static template. The existing assertions already check that the template is rendered and that the errors are displayed in some way.

The addition of the `error_type`s to the mock ingest payload is useful though. Now we know that their inclusion won't cause the template to fail being rendered.